algorithms/CPlusPlus/Arrays/boyer_more.cpp

```diff
@@ -0,0 +1,66 @@
+//majorty element of an array 
+// element with frequency > (n/2) {n: size of array}
+//time complexity : O(n) || space complexity : O(1)
+
+
+#include<iostream>
+#include<stdio.h>
+
+
+using namespace std;
+
+
+void ismaj(int arr[],int n,int exp_maj){
+
+    int i=0;
+    int len = n;
```

Comment:
please add the function description and some comment so it look good

Reply:
yes updating